jobs/resource_job.go

```diff
@@ -906,7 +906,8 @@ func gitSourceSchema(s map[string]*schema.Schema, prefix string) {
 }
 
 func fixWebhookNotifications(s map[string]*schema.Schema) {
```

Comment:
we also need to apply the same sort in `sortWebhookNotifications` function

Reply:
done